src/components/Home/Home.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
- My understanding is this this is a typescript file (specifically one that allows jsx tags versus the `.ts` files) that imports React hooks useState and useEffect
- We have a const variable `Home` that returns html for a Welcome Page